python/psautohint/otfFont.py

```diff
@@ -1014,13 +1012,14 @@ def convertBezToT2(bezString):
 
 
 class CFFFontData:
-    def __init__(self, ttFont, inputPath, outFilePath, fontType, logMsgCB):
+    def __init__(self, ttFont, inputPath, outFilePath, allow_decimal_coords,
+                 font_format, logMsgCB):
```

Comment:
there  a mixture of camelCase (predominant in the original codebase) and snake_case. I guess we should stick to one style or the other.

Reply:
I’m moving away from camelCase as this code is being, slowly, rewritten.